### ISSUE TITLE
crypto.randomUUID is not a function

### ISSUE THREAD

**Issue by zYeoman** [marimo-team/marimo#1724]
### Describe the bug

There is no crypto.randomUUID function without HTTPS, so when I visit marimo running on remote server at http://serverip:2718/, it shows that `crypto.randomUUID is not a function`. Is there any way to get rid of it?

### Environment

```json
{
  "marimo": "0.6.26",
  "OS": "Linux",
  "OS Version": "5.15.0-94-generic",
  "Processor": "x86_64",
  "Python Version": "3.9.19",
  "Binaries": {
    "Browser": "--",
    "Node": "v19.9.0"
  },
  "Requirements": {
    "click": "8.1.7",
    "importlib-resources": "missing",
    "jedi": "0.19.1",
    "markdown": "3.6",
    "pymdown-extensions": "10.8.1",
    "pygments": "2.18.0",
    "tomlkit": "0.12.5",
    "uvicorn": "0.30.1",
    "starlette": "0.37.2",
    "websockets": "12.0",
    "typing-extensions": "4.10.0",
    "black": "22.3.0"
  }
}
```

### Code to reproduce

run `marimo edit llama.py --host 0.0.0.0` on my server, and visit http://serverip:2718/. It shows `crypto.randomUUID is not a function`

**Comment by mscolnick** [comment]
Where does it show that message? Is there a stack trace? I don't think I've seen this message before. 

Can you try it in private browsing or another browser, in case it's one of your extensions.

**Comment by mscolnick** [comment]
This looks like a bug in a down stream dependency - in the meantime, can you visit `http://localhost:2718/` or `https://serverip:2718/` (needs to be development or a https url)

**Comment by mscolnick** [comment]
I am hoping this fixes the issue https://github.com/marimo-team/marimo/pull/1729 we will let you know when we release

**Comment by akshayka** [comment]
@zYeoman , we've released (0.7.0). Let us know if your issue is resolved?

**Comment by zYeoman** [comment]
> @zYeoman , we've released (0.7.0). Let us know if your issue is resolved?

The issue is fixed, thank you.
